src/RMGGeomBench.cc

```diff
@@ -0,0 +1,552 @@
+// Copyright (C) 2025 Moritz Neuberger <https://orcid.org/0009-0001-8471-9076>
```

Comment:
license headers as in all other files, please... (also now in the python files, I added them only recently)